packages/wonder-blocks-tooltip/src/components/__docs__/tooltip.stories.js

```diff
@@ -262,6 +262,40 @@ TooltipOnButtons.parameters = {
     },
 };
 
+/**
+ * Opening a tooltip programatically (Controlled)
+ */
+export const Controlled: StoryComponentType = () => {
+    const [opened, setOpened] = React.useState(false);
+    const buttonText = `Click to ${opened ? "close" : "open"} tooltip`;
+
+    return (
+        <View style={[styles.centered, styles.row]}>
+            <Tooltip
+                content="You opened the tooltip with a button"
+                opened={opened}
+            >
+                tooltip
+            </Tooltip>
+            <Button onClick={() => setOpened(!opened)}>{buttonText}</Button>
+        </View>
+    );
+};
```

Comment:
I had a look at this story in Chromatic and I think it would be better to start the story with `opened` set to `true`.